src/traces/histogram/calc.js

```diff
@@ -196,6 +205,19 @@ function calcAllAutoBins(gd, trace, pa, mainData) {
 
                 binSpec = Axes.autoBin(pos0, pa, tracei['nbins' + mainData], false, calendar);
 
+                // Edge case: single-valued histogram overlaying others
+                // Use them all together to calculate the bin size for the single-valued one
+                if(isOverlay && binSpec._count === 1 && pa.type !== 'category') {
+                    // trace[binAttr] = binSpec;
```

Comment:
:hocho: ?

Reply:
thanks - 🔪  in  https://github.com/plotly/plotly.js/pull/2028/commits/c057d25beb37df646a10136f9ca7781331e647a7